UITests/BrowserTests.swift

```diff
@@ -32,10 +32,15 @@ class BrowserTests: KIFTestCase {
         
         // Show share sheet and wait for the JS prompt to fire
         EarlGrey.select(elementWithMatcher: grey_accessibilityLabel("Share")).perform(grey_tap())
-        let matcher = grey_allOf([grey_accessibilityLabel("Cancel"),
-                                          grey_accessibilityTrait(UIAccessibilityTraitButton),
-                                          grey_sufficientlyVisible()])
-        EarlGrey.select(elementWithMatcher: matcher).perform(grey_tap())
+        
+        if BrowserUtils.iPad() {
+            EarlGrey.select(elementWithMatcher: grey_accessibilityLabel("Share")).perform(grey_tap())
```

Comment:
Oh, it's because there is no cancel button on ipad, simplest way to get rid of the menu was to tap it again